addon/lib/shooter.js

```diff
@@ -551,16 +566,16 @@ const ShotContext = Class({
           this.shot.update(attrs);
         }, this)));
       watchPromise(allPromisesComplete(promises).then((function () {
-        return this.shot.save();
+        //return this.shot.save();
```

Comment:
I guess this whole handler is now unnecessary?  Except you shouldn't be able to save until it completes (which this code currently doesn't protect against)